doc/advanced/PotActions.md

```diff
@@ -36,10 +36,11 @@ override def handle = {
       case PotPending =>
         noChange
       case PotReady =>
-        updated(action.value)
+        updated(action.potResult)
       case PotUnavailable =>
         updated(value.unavailable())
       case PotFailed =>
+        val ex = action.result.failed.get
```

Comment:
`PotFailed` is a case object and doesn't have any parameters. (The `Failed` case class, which extends Pot[Nothing], does have a `Throwable` parameter.) 

When I was trying to get my code to work, I based the above changes on the code in `PotAction.handler`.